.isort.cfg

```diff
@@ -6,9 +6,8 @@ line_length=100
 multi_line_output=3
 include_trailing_comma=True
 force_grid_wrap=0
-use_parentheses=true
-indent=2
-lines_after_imports=2
+use_parentheses=True
```

Comment:
Curious about this case change from `true` to `True`, is this intentional?

Reply:
Yes, it was intentional for consistency with the above value using `=True`. Since `.cfg` is the same thing as `.ini`, we can use either `True` or `true` to mean the same thing.